config.json

```diff
@@ -42,8 +42,8 @@
       "values": ["True", "False"],
       "defaultValue": true
     },
-    "TestArchitecture": {
-      "description": "Sets the architecture value that will be used for testing.",
+    "ArchitectureGroup": {
```

Comment:
Ideally, we'd pass a configuration string.  The difficulty, at the moment, is that we use build.cmd to call both build-native and build-managed.  We don't yet have a way for the native builds to parse a configuration string.  It will take some thought on how to best make this work or work-around this.  For now, I'm passing ArchGroup until we come up with a better solution.